recipes/gromacs/gromacs.py

```diff
@@ -42,9 +42,15 @@
 ######
 # Runtime image stage
 ######
-Stage1 += baseimage(image='nvidia/cuda:10.1-runtime-ubuntu16.04')
+Stage1 += baseimage(image='nvidia/cuda:10.1-base-ubuntu16.04')
 
-Stage1 += Stage0.runtime()
+Stage1 += Stage0.runtime(_from='build')
+
+# Singularity 3.5.3 will not copy symlinks from another stage, so
+# use the full real path to libcufft.
+Stage1 += copy(_from='build',
+               src='/usr/local/cuda/targets/x86_64-linux/lib/libcufft.so.10.1.1.243',
+               dest='/usr/local/cuda/lib64/libcufft.so.10')
```

Comment:
@samcmill Is there any reason to manually copy the library rather than using `apt` to install `cuda-cufft-10-1`?

Reply:
Good idea!